src/components/modals/AddNewCategoryModal.js

```diff
@@ -63,13 +63,12 @@ const AddNewCategoryModal = ({ open, onClose }) => {
   });
 
   const handleSubmit = () => {
-    event.preventDefault();
-
     if (formData.name) {
-      axios
-        .post('http://localhost:5001/tags', {
-          name: formData.name
-        })
+      const tagData = {
+        name: formData.name
+      };
+
+      createTag(tagData)
```

Comment:
It should be passed as `tagName` instead of `name`. You can see the latest property names on the swagger